crm_claim/report/crm_claim_report.py

```diff
@@ -1,11 +1,14 @@
 # Copyright 2015-2017 Odoo S.A.
 # Copyright 2017 Tecnativa - Vicent Cubells
 # Copyright 2018 Tecnativa - Cristina Martin R.
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# Copyright 2020 Tecnativa - Manuel Calero
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 
-from psycopg2.extensions import AsIs
+# pylint: disable=sql-injection
```

Comment:
Is this necessary?

Why not to avoid psycopg2 formatters and use a report style like Odoo does: https://github.com/odoo/odoo/blob/13.0/addons/sale/report/sale_report.py#L61

Reply:
@Yajo  and I, we were looking at several ways to solve it, he recommended that I use that directive.

Reply:
Indeed much better @chienandalu, thanks for the tip.